modules/standard/Math.chpl

```diff
@@ -1057,6 +1057,72 @@ module Math {
     return truncf(x);
   }
 
+
+  /* Returns the Bessel function of the first kind of order `0` of `x`. */
+  extern "j0f" proc j0(x: real(32)): real(32);
+
+  /* Returns the Bessel function of the first kind of order `0` of `x`. */
+  inline proc j0(x : real(64)): real(64) {
+    extern proc j0(x: real(64)): real(64);
+    return j0(x);
+  }
+
+
+  /* Returns the Bessel function of the first kind of order `1` of `x`. */
+  extern "j1f" proc j1(x: real(32)): real(32);
+
+  /* Returns the Bessel function of the first kind of order `1` of `x`. */
+  inline proc j1(x : real(64)): real(64) {
+    extern proc j1(x: real(64)): real(64);
+    return j1(x);
+  }
+
+
+  /* Returns the Bessel function of the first kind of order `n` of `x`. */
+  extern "jnf" proc jn(n: int, x: real(32)): real(32);
+
+  /* Returns the Bessel function of the first kind of order `n` of `x`. */
+  inline proc jn(n: int, x : real(64)): real(64) {
+    extern proc jnf(n: c_int, x: real(64)): real(64);
+    return jnf(n: c_int, x);
+  }
+
+
+  /* Returns the Bessel function of the second kind of order `0` of `x`. */
+  extern "y0f" proc y0(x: real(32)): real(32);
+
+  /* Returns the Bessel function of the second kind of order `0` of `x`. */
```

Comment:
We should mention that `y*` Bessel functions do not support `x < 0` values in the documentation string